nss-day-cohort-48/rare-api-orchestra-outlaws
# VC Register

- added `register_user` to `rare_api/views/auth.py` and corresponding `__init__.py`
- implemented `/register` url with `register_user` function
- created a script to rebuild the database
To Test:
- run `./rebuild_database` from the repo root
- try POSTing a user to `/register` -- must have the following fields:
    - username
    - password
    - first_name
    - last_name
    - email
- verify that a token is returned
